source/install/prepare-mattermost-database.rst

```diff
@@ -204,3 +207,25 @@ When the PostgreSQL database is installed, and the initial setup is complete, yo
 .. important::
 
   If you are upgrading a major version of Postgres, it is essential that ``ANALYZE VERBOSE`` is run on the database post upgrade. This is necessary to re-populate the ``pg_statistics`` table used to generate optimal query plans. The database performance might suffer if this step is not done.
+
+
+Minimum supported version policy
+---------------------------------
+
+To make planning easier and ensure your Mattermost deployment remains fast and secure, we are introducing a policy for updating the minimum supported version of Postgres. The oldest supported Postgres version Mattermost supports will match the oldest version supported by the Postgres community. This ensures you benefit from the latest features and security updates.
+
+This policy change takes effect from Mattermost v10.6, where the minimum Postgres version required will be Postgres 13. This aligns with the Postgres community's support policy, which provides 5 years of support for each major version.
+
+.. note::
+
+  Mattermost v10.6 is not an :ref:`Extended Support Release (ESR) <about/release-policy:extended support releases>`. Going forward, this database version support policy will only apply to ESR releases. When a Postgres version reaches its end of life (EOL), Mattermost will require a newer version starting with the next scheduled ESR release.
+
+Customers will have 9 months to plan, test, and upgrade their Postgres version before the new requirement takes effect. If you need more time, extended ESR support may be available on a case-by-case basis. This policy aims to provide clarity and transparency so you can align database upgrades with the Mattermost release schedule. Contact a `Mattermost Expert <https://mattermost.com/contact-sales/>`_. to discuss your options.
```

Comment:
I think we should remove the "If you need more time..." sentence.  While this *may* be true, I don't think we want to publicize it.